driver/razermouse_driver.c

```diff
@@ -4875,7 +4876,11 @@ struct usb_interface *find_intf_with_proto(struct usb_device *usbdev, u8 proto)
  */
 static struct razer_mouse_device *find_mouse(struct hid_device *hdev)
 {
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 4, 0)
+    const struct bus_type *hid_bus_type = hdev->dev.bus;
+#else
     struct bus_type *hid_bus_type = hdev->dev.bus;
+#endif
```

Comment:
I don't think this has to be guarded, it shouldn't be a problem taking a normal pointer declaring it const here.